src/components/firstProfile/NameProfile.tsx

```diff
@@ -16,7 +16,7 @@ const FULLNAME_REGEX = /^[a-zA-Zа-яА-ЯёЁ\s\p{L}]{2,15}$/u
 const NameProfile = () => {
   const { classes } = useStyles()
   const commonClasses = commonStyles().classes
-  const [fullName, setFullName] = useState('')
+  const [, setFullName] = useState('')
```

Comment:
Why did you remove fullName?

Reply:
it's not used anywhere so far )

Reply:
Do we need to `setFullName`, if `fullName` isn't used?

Reply:
Let's leave it as it is, bc it requires fixes anyway (i approved it to let Sofia moving further)

Reply:
Ok